frontend/src/components/@common/Searchbar/SearchbarInput.tsx

```diff
@@ -0,0 +1,41 @@
+import { ComponentPropsWithRef, forwardRef } from 'react';
+import styled from 'styled-components';
+import SearchIcon from '~/assets/icons/search.svg';
+import { FONT_SIZE } from '~/styles/common';
+
+interface SearchbarInputProps extends ComponentPropsWithRef<'input'> {
+  width: number;
+}
+
+const SearchbarInput = forwardRef<HTMLInputElement, SearchbarInputProps>(({ width, onChange }, ref) => (
+  <StyledSearchbarInputContainer width={width}>
+    <SearchIcon />
+    <StyledSearchbarInput type="text" ref={ref} onChange={onChange} />
+  </StyledSearchbarInputContainer>
+));
+
+export default SearchbarInput;
+
+const StyledSearchbarInputContainer = styled.div<{ width: number }>`
+  display: flex;
+
+  width: ${props => `${props.width}px`};
```

Comment:
당연히 알수도 있지만 이렇게도 표현해 볼 수 있어요!

```suggestion
  width: ${({width}) => `${width}px`};
```

Reply:
> 수정했습니다!!👍👍